scripts/refresh-pr-10841.py

```
@@ -0,0 +1,390 @@
+
+        expect(aggregateDragOverNode.onDragDrop).toHaveBeenCalledWith(event)
+        expect(positionNode.onDragDrop).not.toHaveBeenCalled()
+      } finally {
+        canvasContainer.remove()
+      }
+    })
+"""
+    tail = "\n  })\n})\n"
+    index = text.rfind(tail)
+    if index < 0:
+        raise RuntimeError('final describe tail not found')
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)